test/e2e/apimachinery/validatingadmissionpolicy.go

```diff
@@ -356,6 +359,16 @@ var _ = SIGDescribe("ValidatingAdmissionPolicy [Privileged:ClusterAdmin]", func(
 					return false, err
 				}
 				if policy.Status.TypeChecking != nil {
+					// TODO(#123829) Remove once the schema watcher is merged.
```

Comment:
Since type-checking is best-effort / advisory / non-blocking, I guess deflaking this way can make sense, but I don't think this e2e test should be a conformance test in its current state.

I'd recommend switching the `should type check a CRD` e2e test back to a normal test, not a conformance test